python/cudf/cudf/core/index.py

```diff
@@ -579,7 +579,7 @@ def any(self):
         """
         return self._values.any()
 
-    def to_pandas(self):
+    def to_pandas(self, **kwargs):
```

Comment:
Do we need to pass the kwargs into the below `to_pandas()` call? Do we allow nulls in the index?

Reply:
Actually this is not needed. Initially added them with a view that pandas supports nullable dtypes in Multi-Index. But it appears to not be the case:

```python
In[19]: df = pd.DataFrame()
In[20]: df['a'] = pd.Series([1, 2, 3, None, 1, 3], dtype="Int64")
In[21]: df
Out[21]: 
      a
0     1
1     2
2     3
3  <NA>
4     1
5     3
In[22]: df['b'] = pd.Series([1, 2, None, None, 434, 3], dtype="Int64")
In[23]: df['c'] = pd.Series(['asd','ds','sdff','dsf','sdf']) 
In[24]: df
Out[24]: 
      a     b     c
0     1     1   asd
1     2     2    ds
2     3  <NA>  sdff
3  <NA>  <NA>   dsf
4     1   434   sdf
5     3     3   NaN
In[25]: df.set_index(['a','b'])
Out[25]: 
            c
a   b        
1   1     asd
2   2      ds
3   NaN  sdff
NaN NaN   dsf
1   434   sdf
3   3     NaN
In[26]: df.set_index(['a','b']).index
Out[26]: 
MultiIndex([(1.0,   1.0),
            (2.0,   2.0),
            (3.0,   nan),
            (nan,   nan),
            (1.0, 434.0),
            (3.0,   3.0)],
           names=['a', 'b'])
```